postcss.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
       features: {
         'nesting-rules': true,
         'focus-visible-pseudo-class': false,
+        'logical-properties-and-values': false,
```

Comment:
Had to disable `logical-properties-and-values` to make properties like `padding-inline-start` work.

When set to `true`, the output adds `[dir="ltr"]` + `[dir="rtl"]` to the selector, I assume as a fallback. But we currently don't use these attributes in Lookbook/dotcom.